content/docs/actors/development/output_schema.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Output schema
+description: Output schema is designed to help actor creators present the results in an output UI attractive to actor users.
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actors/development/output-schema
+---
+
+# Actor output schema
+
+ It is recommended to show the most important fields in a curated Overview visualization configured using output schema specification, while all available fields are automatically available in the “All fields” view.
+
+In the future, output schema will also help with strict output data format validation, which will make integrations more solid and easier to set up.
+
+## Specification version 1
+
+An actor's output schema defines the structure and both API and visual representation of data produced by an actor. Output configuration files have to be located in the `.actor` folder in the actor's root directory.
+
+## How to organize files in the .actor folder: two options
+
+**A)** all config options are being set in a .actor/actor.json file, e.g.:
+
+```JSON
+//file: .actor/actor.json
+{
+    "actorSpecification": 1,
+    "name": "this-is-book-library-scraper",
+    "title": "Book Library scraper",
+    "version": "1.0.0",
+    "storages": {
+        "dataset": {
+            "actorSpecification": 1,
+            "fields": {},
+            "views": {
+                "overview": {
+                    "title": "Overview",
+                    "transformation": {},
+                    "display": {}
+                }
+            }
+        }
+    }
+}
+```
+
+**B)** .actor/actor.json links to other sub-config files in the same folder, e.g.:
+
+```JSON
+//file: .actor/actor.json
```

Comment:
Do we need these when you already state the file name above?

Reply:
I would rather keep it there for users that are copy-pasting code, to help them put the code in the correct file.